crates/uv/tests/pip_list.rs

```diff
@@ -449,41 +329,25 @@ fn list_exclude() -> Result<()> {
     exit_code: 0
     ----- stdout -----
     Package Version
-    ------- -------
-    anyio   4.3.0
-    idna    3.6
+    [UNDERLINE]
+    anyio 4.3.0
+    idna 3.6
     sniffio 1.3.1
```

Comment:
e.g. this is a worse snapshot, but it's only one line of filters to maintain and I'm not sure if the old snapshot really tested for alignment when it was doing all those manipulations.